randovania/bitpacking/construct_pack.py

```diff
@@ -16,23 +16,25 @@
 if typing.TYPE_CHECKING:
     from _typeshed import DataclassInstance
 
+    from randovania.lib.construct_stub import CodeGen
```

Comment:
Uh, that doesn't exist?
And Mypy still fails because of this.

Reply:
Ugh forgot to add the file